source/loader/layers/sanitizer/asan_shadow_setup.cpp

```diff
@@ -121,6 +121,60 @@ ur_result_t DestroyShadowMemory() {
 
 } // namespace pvc
 
+namespace dg2 {
+///
+/// USM Allocation Range
+///   Host/Shared USM : 0x0000_0000_0000_0000 ~ 0x0000_7fff_ffff_ffff
+///   Device      USM : 0xffff_8000_0000_0000 ~ 0xffff_ffff_ffff_ffff
```

Comment:
@wenju-he, this is the USM allocation range of DG2, I afraid there's no document. I confirmed this summary from level zero team.